2025.2/config.json

```diff
@@ -0,0 +1,60 @@
+{
+  "defaultFiles": "./defaultFiles",
+  "parentComponents": [
+    {
+      "label": "Static Private App",
+      "type": "app",
+      "authType": "static",
+      "distribution": "private",
+      "path": "components/app/static-private"
+    },
+    {
+      "label": "OAuth Private App",
+      "type": "app",
+      "authType": "oauth",
+      "distribution": "private",
+      "path": "components/app/oauth-private"
+    },
+    {
+      "label": "OAuth Marketplace App",
+      "type": "app",
+      "authType": "oauth",
+      "distribution": "marketplace",
+      "path": "components/app/oauth-marketplace"
+    }
+  ],
+  "components": [
+    {
+      "path": "components/cards",
+      "label": "Card",
+      "type": "card",
+      "parentType": "app",
+      "supportedAuthTypes": ["oauth", "static"],
+      "supportedDistributions": ["private", "marketplace"]
```

Comment:
We use these supported options to determine which components are valid options on `create` and `add`